test/e2e/storage/testsuites/fsgroupchangepolicy.go

```diff
@@ -206,10 +208,17 @@ func (s *fsGroupChangePolicyTestSuite) DefineTests(driver storageframework.TestD
 		test := t
 		testCaseName := fmt.Sprintf("(%s)[LinuxOnly], %s", test.podfsGroupChangePolicy, test.name)
 		ginkgo.It(testCaseName, func() {
+			dInfo := driver.GetDriverInfo()
+			policy := v1.PodFSGroupChangePolicy(test.podfsGroupChangePolicy)
+
+			if dInfo.Capabilities[storageframework.CapVolumeMountGroup] &&
+				utilfeature.DefaultFeatureGate.Enabled(features.DelegateFSGroupToCSIDriver) &&
```

Comment:
I do not think this feature gate check works. I don't think we have an api to detect which feature gates are enabled on a cluster. I think we need to leave it up to the provider to appropriately set the driver capabilities.

Reply:
uh yeah. I thought we were solely relying on `CapVolumeMountGroup`, didn't notice feature gate check.

Reply:
Right, in the (common?) case of e2e binary being the same minor version as the cluster this might help. But if a lower version test binary runs against a higher version cluster, this test will not skip even if the driver has the right capability. So I'll remove this check.

Reply:
https://github.com/kubernetes/kubernetes/pull/106435

Reply:
Even if the e2e.test and the Kubernetes version match exactly you cannot use `utilfeature` in the client to check for features in the server.